pkg/master/master.go

```diff
@@ -564,6 +583,16 @@ func (m *Master) init(c *Config) {
 	requestInfoResolver := &apiserver.APIRequestInfoResolver{util.NewStringSet(strings.TrimPrefix(defaultVersion.Root, "/")), defaultVersion.Mapper}
 	apiserver.InstallServiceErrorHandler(m.handlerContainer, requestInfoResolver, apiVersions)
 
+	if m.exp {
```

Comment:
Is there a reason why this duplicated code can't be factored out into a helper function?